backup/backup_client.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash                                                                                                                                                 
+
+set -euo pipefail
+
+if [ "${1:-}" = "backup-db" ] ; then
+    PGPASSWORD=${DB_PASS} pg_dump -h ${DB_HOST} -p ${DB_PORT} -U ${DB_USER}
```

Comment:
Not sure if DB_HOST, DB_PORT or DB_USER may contain whitespace, but DB_PASS for sure can.

```suggestion
    PGPASSWORD="${DB_PASS}" pg_dump -h "${DB_HOST}" -p "${DB_PORT}" -U "${DB_USER}"
```